notebooks/en/agents.ipynb

```diff
@@ -35,7 +35,7 @@
    "metadata": {},
```

Comment:
<p>huggingface_hub comes with transformers and datasets so we could declutter</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
ah no, you are supposed to remove `huggingface_hub` only

Reply:
My bad, I misunderstood the message. I've restored `datasets` and removed `huggingface_hub`.